frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -43,24 +46,6 @@ const AdminJobsPage = () => {
             [],
             { refetchInterval: refreshJobsIntervalMilliseconds }
         );
-    // Stryker enable  all 
-
-    const jobLaunchers = [
-        {
-            name: "Update Cow Health",
-            form: <JobComingSoon />
-        },
-        {
-            name: "Milk The Cows",
-            form: <JobComingSoon />
-        },
-        {
-            name: "Instructor Report",
-            form: <JobComingSoon />
-        },
-    ]
-
```

Comment:
Please do not remove this, and refactor in this way.  The change you have made doesn't make the code better; only longer and harder to maintain.